src/components/Dashboard/Carousel/Carousel.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Carousel = ({ children }) => {
+  return (
+    <div className="dp-carousel" id="carousel1">
+      <div className="dp-carousel-wrapper dp-carousel-orange">
```

Comment:
I think the color here should be parametrized somehow (`dp-carousel-orange`) it could be set as a prop. 
Also the id should be something that can be set, else we will have more than one carousel with the same id.

Reply:
ok!